pytest_splunk_addon/splunk.py

```diff
@@ -759,13 +760,22 @@ def splunk_ingest_data(request, splunk_hec_uri, sc4s, uf, splunk_events_cleanup)
             )
             sleep(50)
         except Exception as e:
+            with open("ingestion_error_main_worker.txt", "w") as f:
```

Comment:
shouldn't this file name be unique, composed somewhat like `os.environ.get("PYTEST_XDIST_TESTRUNUID") + "_ingestion_error"`? I imagine using a static file name here could lead to a leftover file from previous run (if for example they fail to clean up after themselves) being interpreted as current test failure.